website/docs/r/sagemaker_device_fleet.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+subcategory: "Sagemaker"
+layout: "aws"
+page_title: "AWS: aws_sagemaker_device_fleet"
+description: |-
+  Provides a Sagemaker Device Fleet resource.
+---
+
+# Resource: aws_sagemaker_device_fleet
+
+Provides a Sagemaker Device Fleet resource.
+
+## Example Usage
+
+### Basic usage
+
+```terraform
+resource "aws_sagemaker_device_fleet" "example" {
+  device_fleet_name = "example"
+  role_arn          = aws_iam_role.test.arn
+
+  output_config {
+    s3_output_location = "s3://${aws_s3_bucket.test.bucket}/prefix/"
```

Comment:
Nit: we prefer `example` over `test` in documentation

Reply:
fixed